### ISSUE TITLE
Cant update 

### ISSUE THREAD

**Issue by dimitrow7** [fulder/iocage-plugin-bazarr#1]
Hello,

How to update to newer version? I need version Ver. 0.8.4.3, because of new subtitle source, that I need is added.

When I try with interface, I got this error:

```
Job "Update Bazarr from source on Github (trigger: interval[6:00:00], next run at: 2020-04-14 13:04:19 EEST)" raised an exception.

Traceback (most recent call last):
 File "/usr/local/bazarr/bazarr/../libs/apscheduler/executors/base.py", line 125, in run_job
  retval = job.func(*job.args, **job.kwargs)
 File "/usr/local/bazarr/bazarr/check_update.py", line 44, in check_and_apply_update
  g.fetch(\origin\)
 File "/usr/local/bazarr/bazarr/../libs/git/cmd.py", line 548, in
  return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
 File "/usr/local/bazarr/bazarr/../libs/git/cmd.py", line 1014, in _call_process
  return self.execute(call, **exec_kwargs)
 File "/usr/local/bazarr/bazarr/../libs/git/cmd.py", line 825, in execute
  raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd(\/usr/local/bin/git\) failed due to: exit code(128)
 cmdline: /usr/local/bin/git fetch origin
 stderr: \fatal: \origin\ does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.\
```

**Comment by fulder** [comment]
Hey,

I just realized this plugin was installed by just downloading the tar artifact from the bazarr github releases. This way git didn't get initialized properly and the auto bazarr updater (which assumes it is installed using git) won't work while trying to fetch the changes. I've just merged https://github.com/fulder/iocage-plugin-bazarr/pull/2 to master which should probably fix this issue.

**Comment by dimitrow7** [comment]
Thanks. What I need to do now, to make this work?

**Comment by dimitrow7** [comment]
Quick fix - Backup data directory, remove /usr/local/bazarr/ directory and git clone again. Then restore data dir and restart.

**Comment by fulder** [comment]
Good finding and quickfix. I've merged back master to the `11.3-RELEASE` branch now but not quite sure how the update process for freenas plugins should work. It would work to just reinstall the plugin but then you have to reconfigure it all over again.

**Comment by fulder** [comment]
Added a `post_update.sh` with a migration (from [0.0.1](https://github.com/fulder/iocage-plugin-bazarr/releases/tag/0.0.1) to [0.1.0](https://github.com/fulder/iocage-plugin-bazarr/releases/tag/0.1.0)) with the needed `git` fixes. (And a `pre_update.sh` in 0.2.0)

The migration itself actually fetches the latest from master bazarr repo which will make it update to the latest during the upgrade. And all future updates should probably work better too. Checking bazarr logs I got 

```
2020-04-15 23:12:39,171 - root                             (80de20800) :  INFO (check_update:48) - BAZARR No new version of Bazarr available.
```

Instead of the previous git crash.

@dimitrow7 If you could test test upgrading your plugin as well (it should probably work both using the UI or running `iocage bazarr update` in freenas SSH) and verify if it works as intended it would be much appreciated :)

Thanks again for finding and reporting this bug! :)

**Comment by josdion** [comment]
@fulder Bazarr needs unrar to extract subtitles from rar files. Looks like you removed it in the last commit to bazarr.json.

Bazarr also needs ffprobe for analyzing video files. Probably ffmpeg shoud also be added as a dependency.

**Comment by fulder** [comment]
@josdion good point, I've opened a new PR (https://github.com/ix-plugin-hub/iocage-plugin-index/pull/67) for both `unrar` and `ffmpeg` packages. Maybe these requirments should be added somewhere in the bazarr README or wiki. Couldn't find them anywhere except the Docker images.

**Comment by josdion** [comment]
Generally bazarr doesn't require these, because they are included in bazarr/bin, but only for mac, windows and linux. These dependencies are currently required only for freebsd.

**Comment by fulder** [comment]
Ah, I see! Then it's important to add them to the `bazarr.json`, hopefully it will get merged fast. Thanks for the tip and explanation :)

**Comment by fulder** [comment]
The PR has been merged and back ported to 11.3 branch.

UPDATE: Closing this again.
